spec/features/cart/checkout_spec.rb

```diff
@@ -20,13 +20,27 @@
 
     it 'Theres a link to checkout' do
       visit "/cart"
-
+      login_user
       expect(page).to have_link("Checkout")
 
       click_on "Checkout"
 
       expect(current_path).to eq("/orders/new")
     end
+    describe 'When I am not logged in and visit cart' do
```

Comment:
ignore this for now, this is part of user story 26 that I was in the middle of. Unrelated to edits